src/router.js

```diff
@@ -97,4 +97,13 @@ export default class Router {
   forward() {
     window.history.forward();
   }
+
+  pop() {
+    if (this.currentPath() !== '/') {
```

Comment:
I was going to handle the lack of trailing slash case, but I saw another test that ensured currentPath returns a trailing slash, so I assumed it.

Do you think `/test/` shouldn't go to `/`?